app/components/index.js

```diff
@@ -0,0 +1,5 @@
+export { default as DescriptionInput } from './DescriptionInput/DescriptionInput';
+export { default as ExpirationInput } from './ExpirationInput/ExpirationInput';
+export { default as ListDivider } from './ListDivider/ListDivider';
+export { default as LocationCard } from './LocationCard/LocationCard';
+export { default as LocationsList } from './LocationsList/LocationsList';
```

Comment:
This file serves to bundle all the components together (known as the barrel pattern). That way I can write this:
```javascript
import { DescriptionInput, ExpirationInput, ListDivider, LocationsList } from '../../components';
```
This is much better than having to import each component from its individual file.